oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -168,7 +168,7 @@ public class VersionGarbageCollector {
      * Ultimately the goal is to clean up all possible garbage. After hardening these modes
      * might no longer be supported.
      */
-    enum FullGCMode {
+     enum FullGCMode {
```

Comment:
nitpick, detail, but I would remove this, it is an unnecessary change

Reply:
Fixed, thank you for the comment.